src/components/MDX/Sandpack/template.ts

```diff
@@ -0,0 +1,54 @@
+export const template = {
```

Comment:
This is pretty much the same template initially defined (and now removed) on Sandpack component